src/vibenix/tools/view.py

```diff
@@ -25,7 +25,17 @@ def _view(view_range: list[int]=None) -> str:
         # Check if view_range is valid
         lines = current_content.splitlines()
         # Add `<line_number>: ` prefix to each line
-        lines = [f"{i+1:>3}: {line}" for i, line in enumerate(lines)]
+        from vibenix.defaults.vibenix_settings import get_settings_manager
+
+        # Show line numbers if prompt uses edit tools or if specifically requested
```

Comment:
Does it ever make sense to specifically request line numbers, if the edit tools are not enabled?